include/seqan3/contrib/charconv/charconv.cpp

```diff
@@ -176,25 +176,7 @@ __u64toa(uint64_t value, char* buffer)
         const uint32_t v0 = static_cast<uint32_t>(value / 100000000);
         const uint32_t v1 = static_cast<uint32_t>(value % 100000000);
 
-        const uint32_t b0 = v0 / 10000;
-        const uint32_t c0 = v0 % 10000;
-
-        if (v0 < 1000000)
-        {
-            if (v0 < 100000)
-                buffer = append1(buffer, b0);
-            else
-                buffer = append2(buffer, b0);
-        }
-        else
-        {
-            if (v0 < 10000000)
-                buffer = append3(buffer, b0);
-            else
-                buffer = append4(buffer, b0);
-        }
-
-        buffer = append4(buffer, c0);
+        buffer = __u32toa(v0, buffer);
```

Comment:
Isn't v0 a prefix for all numbers that have more digits than `100000000`? What happens for small numbers like `10`, where `v0` is zero and that prefix should be "" (empty)?

Reply:
This part of the function is only for values in `[100000000, 10000000000000000)`

Reply:
The if body starting at 130 will be used if the value is smaller than `10000000000000000`

Reply:
Ah, I haven't seen that. Thank you.